src/components/common/Chart/ProgressChart.tsx

```diff
@@ -7,11 +7,13 @@ type ProgressChartProps = {
 export default function ProgressChart({ value, target }: ProgressChartProps): JSX.Element {
   const theme = useTheme();
 
+  const percentage = (100 * value) / target;
+
   return (
     <LinearProgress
       variant='determinate'
-      value={value}
-      valueBuffer={target}
+      value={percentage}
```

Comment:
I'm using this chart for HectaresPlantedCard. Should it be updated?

Reply:
The change should be transparent to the consumer.

Reply:
Thank you!